Automattic/pocket-casts-android
# Open app if widget is taped outside of the buttons

## Description

This PR changes widget behavior so that the app is opened when anything outside of the button is tapped.

## Testing Instructions

1. Add widgets to the home screen.
2. When you have nothing playing tapping on anything in the widgets should open the app.
3. When you have something playing tapping outside of the buttons should open the app and tapping on the buttons should control the playback.

## Checklist
 
#### I have tested any UI changes...
